addons/microsoft_outlook/views/ir_mail_server_views.xml

```diff
@@ -5,34 +5,30 @@
         <field name="model">ir.mail_server</field>
         <field name="inherit_id" ref="base.ir_mail_server_form"/>
         <field name="arch" type="xml">
-            <field name="smtp_host" position="before">
-                <field name="use_microsoft_outlook_service" string="Outlook"/>
-            </field>
             <field name="smtp_user" position="after">
                 <field name="is_microsoft_outlook_configured" invisible="1"/>
                 <field name="microsoft_outlook_refresh_token" invisible="1"/>
                 <field name="microsoft_outlook_access_token" invisible="1"/>
                 <field name="microsoft_outlook_access_token_expiration" invisible="1"/>
-                <div></div>
-                <div attrs="{'invisible': [('use_microsoft_outlook_service', '=', False)]}">
-                    <span attrs="{'invisible': ['|', ('use_microsoft_outlook_service', '=', False), ('microsoft_outlook_refresh_token', '=', False)]}"
+                <div attrs="{'invisible': [('smtp_authentication', '!=', 'outlook')]}"
+                    class="d-flex flex-row align-items-center">
```

Comment:
That's quite uncommon to use custom styling in form views, can't it be done using standard form rendering ?

Reply:
we can, but it won't be 100% centered with standard "row" class